packages/create-plugin/src/commands/generate-actions/update-go-sdk-and-packages.ts

```diff
@@ -6,34 +6,57 @@ import { exec } from 'child_process';
 
 const SDK_GO_MODULE = 'github.com/grafana/grafana-plugin-sdk-go';
 
-export function updateGoSdkAndModules({ pluginName, orgName, pluginType }: CliArgs): Promise<string> {
-  return new Promise(async (resolve, reject) => {
-    // plugin directory
-    const exportPath = getExportPath(pluginName, orgName, pluginType);
+export async function updateGoSdkAndModules({ pluginName, orgName, pluginType }: CliArgs): Promise<string> {
```

Comment:
nit: regardless of the user choosing a backend or not this function is run and prints `updateGoSdkAndModules Grafana go sdk updated successfully.` which is a little misleading as it does nothing without the backend parts. Probably best in a future PR to only run it when `hasBackend` is `true`. WDYY?